app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    def index
+      @pagy, @comments = pagy(Comment.all)
```

Comment:
Woo pagy!

Reply:
I'm going to do some research on how to actually test performance for this :) Shouldn't be too complicated to switch other controllers/views to this just time and 👀 .

Reply:
OMG, I'm too excited. I'd be curious just to see what kinds of requests it makes (as seen in logs)

Reply:
Well, for starters :) we'd go from this:
```
  <div id="left" class="comments_area box">
    <h1>Recent Comments</h1>
    <div class="mini_paginator">
      <%= will_paginate @comments, param_name: :page%>
    </div>
    <%= render partial: 'shared/comment', collection: @comments %>
    <div class="mini_paginator">
      <%= will_paginate @comments, param_name: :page%>
    </div>
  </div>
```
to this:
```
  <div id="left" class="comments_area box">
    <h1>Recent Comments With Pagy</h1>
    <%== pagy_nav @pagy %>
    <%= render partial: 'shared/comment', collection: @comments_for_pagy %>
    <%== pagy_nav @pagy %>
  </div>
```

Reply:
Don't quote me on that, but here is what I did:
- setup some caching in my `development.rb` (it seems like pagy's benefits increase quite a lot once caching is enabled)
- performed simple Rails view benchmarking test 
Hard reloads: 
```
[22:35:16] (pagy-test) alonetone
$ tail -f log/development.log | grep 'Comments'
Processing by CommentsController#index as HTML
Comments (4156.4ms)
Comments with pagy (141.5ms)
Processing by CommentsController#index as HTML
Comments (1271.1ms)
Comments with pagy (111.4ms)
Processing by CommentsController#index as HTML
Comments (3525.9ms)
Comments with pagy (1504.4ms)
Processing by CommentsController#index as HTML
Comments (1607.2ms)
Comments with pagy (89.4ms)
Processing by CommentsController#index as HTML
Comments (376.9ms)
Comments with pagy (115.0ms)
```
Soft reloads
```
[22:36:52] (pagy-test) alonetone
$ tail -f log/development.log | grep 'Comments'
Processing by CommentsController#index as HTML
Comments (2595.4ms)
Comments with pagy (91.5ms)
Processing by CommentsController#index as HTML
Comments (1567.8ms)
Comments with pagy (188.0ms)
Processing by CommentsController#index as HTML
Comments (3659.9ms)
Comments with pagy (1039.4ms)
Processing by CommentsController#index as HTML
Comments (2545.5ms)
Comments with pagy (199.1ms)
Processing by CommentsController#index as HTML
Comments (643.9ms)
Comments with pagy (140.0ms)
Processing by CommentsController#index as HTML
Comments (2331.8ms)
Comments with pagy (103.9ms)
```